packages/myst-cli/src/process/file.ts

```diff
@@ -50,7 +48,7 @@ export async function loadFile(
   file: string,
   projectPath?: string,
   extension?: '.md' | '.ipynb' | '.bib',
-  opts?: { minifyMaxCharacters?: number; preFrontmatter?: Record<string, any> },
```

Comment:
I want to make sure this still works as expected. I don't now see where the minify code is called?

Reply:
See below for comments targe
tting the appropriate lines.